signingscript/docker.d/passwords.yml

```diff
@@ -206,7 +206,7 @@ in:
           - ["https://autograph-external.prod.autograph.services.mozaws.net",
              {"$eval": "AUTOGRAPH_FENNEC_NIGHTLY_USERNAME"},
              {"$eval": "AUTOGRAPH_FENNEC_NIGHTLY_PASSWORD"},
-             ["autograph_apk"]
+             ["autograph_apk_fennec_sha1"]
```

Comment:
This value was taken from https://github.com/mozilla-releng/scriptworker-scripts/blob/3da24fc4d0ef5c8de521bbcd082dd1ed80c4a7f1/signingscript/docker.d/passwords.yml#L149